### PR TITLE
Adding timezone.js to address issue #18

### PR DESCRIPTION
**Pull request by TomStone95**
1. Updating package.json to get rid of outdated xvfb-run
2. There are two commits for taskbot.js ("added test/call response" and "changing back taskbot.js") No changes were made.
3. Adding timezone.js to address issue #18

### CODE REVIEW

**Comment by TomStone95** [comment]
@tiagosiebler 
Let me know if this is what you had in mind. I think it's a good start, but I think I'm also ready to move onto a different issue.

File: package.json [hunk 1]
```diff
@@ -5,7 +5,7 @@
   "main": "bot.js",
   "scripts": {
     "start1": "node bot.js",
-    "start": "xvfb-run node bot.js",
```
Comment (tiagosiebler) [inline package.json]: Hey Tom, please bring back `xvfb-run`. The prod env is headless, which prevents the scraper from working as it does on a dev laptop, unless xvfb-run is used:
https://github.com/tiagosiebler/Jarvis/blob/master/skills/search.js#L2

File: skills/timezone.js [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+    const getTimeForTimezone = function (timezone){
+        var date = new Date();
+        let hours = date.getHours();
+        let adjustedHours = hours + adjustForTimezone(timezone);
+        let minutes = (date.getMinutes()<10?'0':'') + date.getMinutes();
+        return adjustedHours + ':' + minutes;
+    }
+
+    //time zone adjustment
+    const adjustForTimezone = function (timezone){
```
Comment (tiagosiebler) [inline skills/timezone.js]: Any ideas on generalising this so it applies on other non-US timezones?

**Review by tiagosiebler — CHANGES_REQUESTED** [review]
Nice work! Some requests. Please also run this command:
```
npx eslint --fix skills/timezone.js
```

This automates much of the formatting to fit with the rest of the repo.

File: skills/timezone.js (hunk 2 above)
Comment (TomStone95) [reply skills/timezone.js]: Either hardcoding all of the timezones (expanding on what i currently have) or hitting some sort of api. I can't think of any in-between solution.

File: package.json (hunk 1 above)
Comment (TomStone95) [reply package.json]: Whoops....will do!

**Review by TomStone95 — COMMENTED** [review]
I think I'm done! Let me know if anything else is needed

**Pull request merged**